src/main/java/org/mockito/internal/session/DefaultMockitoSessionBuilder.java

```diff
@@ -1,23 +1,51 @@
 /*
- * Copyright (c) 2017 Mockito contributors
+ * Copyright (c) 2018 Mockito contributors
  * This program is made available under the terms of the MIT License.
  */
 package org.mockito.internal.session;
 
 import org.mockito.MockitoSession;
 import org.mockito.internal.framework.DefaultMockitoSession;
 import org.mockito.internal.util.ConsoleMockitoLogger;
+import org.mockito.internal.util.MockitoLogger;
 import org.mockito.quality.Strictness;
 import org.mockito.session.MockitoSessionBuilder;
+import org.mockito.session.MockitoSessionLogger;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.UUID;
+
+import static java.util.Collections.emptyList;
 
 public class DefaultMockitoSessionBuilder implements MockitoSessionBuilder {
 
-    private Object testClassInstance;
+    private List<Object> testClassInstances = new ArrayList<Object>();
+    private String name;
     private Strictness strictness;
+    private MockitoSessionLogger logger;
 
     @Override
     public MockitoSessionBuilder initMocks(Object testClassInstance) {
-        this.testClassInstance = testClassInstance;
+        if (testClassInstance != null) {
+            this.testClassInstances.add(testClassInstance);
+        }
+        return this;
+    }
+
+    @Override
+    public MockitoSessionBuilder initMocks(Object... testClassInstances) {
+        if (testClassInstances != null) {
+            for (Object instance : testClassInstances) {
+                initMocks(instance);
```

Comment:
Few questions:
- why this method initializes the mocks but the non-vararg method just appends to the list?
- why do we need 2 methods: the vararg and non-vararg initMocks?

Reply:
It doesn't initialize the mocks, it calls the `initMocks(Object)` method on the builder to not have to duplicate the null check.

Reply:
Got it. Thank you for clarifying. I misread the code.